src/Components/WebAssembly/Build/src/targets/CollationLinkerDescriptor.xml

```diff
@@ -0,0 +1,21 @@
+<linker>
+  <!-- This file disables exclusions that remove collation data -->
```

Comment:
It's the opposite now isn't it? This file now *does* remove the collation data.

BTW please don't feel the need to rebuild this PR just for minor tweaks :) We can correct comments later.